contributing_to_docs/tools_and_setup.adoc

```diff
@@ -65,6 +65,7 @@ install the software and tools you will use to create the content. All OpenShift
 documentation is created in AsciiDoc, and is processed with https://github.com/redhataccess/ascii_binder[AsciiBinder],
 which is an http://asciidoctor.org/[AsciiDoctor]-based docs management system.
 
+NOTE: If you do not want to install the tools directly onto your system, you can use `build.sh` to run the tools in a container via podman.
```

Comment:
Are there other requirements besides having podman installed? I installed podman and started the container, as it suggested, but can't get this to work.

Reply:
You should need anything else, but it's possible I've made some assumptions. What OS are you on (it probably only works on Fedora, CentOS, or RHEL)? Can you share the output you're seeing? (gist, slack, whatever's easy)

Reply:
I'm on mac. I assumed that because I could install podman that this would work. :)

Reply:
The latest update worked somewhat on my M1 Mac. There were some warnings about include files and images that binder couldn't read but that are definitely present. I'm testing on x86 now.

Reply:
I saw similar errors on x86, so I updated the disclaimer to suggest that it does not work on macOS, yet.